### ISSUE TITLE
ULS UNITS ASIA - Fix keyboard interactions with dropdown menus

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#457]
* Keyboard cannot interact with 2nd level drop downs in the main navigation (or lower)
* When the keyboard leaves a drop down, the drop down should disappear

**Comment by kevinkidwell** [comment]
@manolobevia I've been trying to get this solved on Asia but I can't seem to get every part of it working. So far I'm able to interact with level 2 and below but then I can't get any of the menus to close when you tab out of them. Do you have any ideas that might work?

**Comment by manolobevia** [comment]
@kevinkidwell I haven't developed Asia library site. I provided static markup templates to the developer, but what it is right now it is completely different to what I did. The use of a tree menu was a bad choice for the developer. There are alternatives that can help the user get their faster and in a more pleasurable way as you know.

This is insane:
![Screen Shot 2019-09-10 at 11 18 36 AM](https://user-images.githubusercontent.com/8395482/64642751-0e243d80-d3dd-11e9-879f-e00194a68ace.png)

This was the original design, which has just 2 level navigation and a better IA.
![Screen Shot 2019-09-10 at 11 41 04 AM](https://user-images.githubusercontent.com/8395482/64642802-25632b00-d3dd-11e9-99a6-7aaaff898318.png)

Let me check the menu module the developer used and see if I can fix the issue. Also, I have noticed the menu on mobile doesn't display links, probably due to color changes to improve contract.

![Screen Shot 2019-09-10 at 11 26 31 AM](https://user-images.githubusercontent.com/8395482/64642947-65c2a900-d3dd-11e9-86bb-704db3783424.png)

**Comment by kevinkidwell** [comment]
@manolobevia I can look into the mobile menu color. If we can just get the navigation to work with the keyboard as CIT needs it, this site is only going to be around for another semester and then it'll be migrated to WP and we can narrow down these menus to a user-friendly number.

**Comment by manolobevia** [comment]
@kevinkidwell I didn't want it to give you the impression we should change the main navigation now. When we migrate we should change it.
